app.js

```diff
@@ -2,15 +2,48 @@
 // import { getRandomItem } from './utils.js';
 
 /* State */
+let gameState = 'guess'; // 'guess' or 'results'
+let guess = ''; // 'shell-1', 'shell-2', or 'shell-3'
+let reveal = ''; // 'pearl-1', 'pearl-2', or 'pearl-3'
+let results = ''; // 'Win!' or 'Lose'
+
+let lift = 0;
+let wins = 0;
 
 /* Actions */
-function loadPage() {}
+function loadPage() {
+    displayShells();
+    displayResults();
+    displayScoreboard();
```

Comment:
you're calling several functions on pageload but you haven't said what those functions do. part of working vertically is picking one function at a time and getting that working before creating the next one.

if you want to use that space for planning you can use pseudocode: 
```
function loadPage() {
    // render shells on page
    // then render results
    // then render the scoreboard
}
```
then taking each comment and building out that function